db/merge_helper.cc

```diff
@@ -414,7 +413,9 @@ CompactionFilter::Decision MergeHelper::FilterMerge(const Slice& user_key,
                                                        kValueTypeForSeek);
     }
   }
-  total_filter_time_ += filter_timer_.ElapsedNanosSafe();
+  if (ShouldReportDetailedTime(env_, stats_)) {
```

Comment:
I know it does not matter for execution purposes, but can you make this "ShouldReportTime" condition match the earlier one [ If (stats_ != nullptr && ... ) ]

Otherwise, someone is likely to look later and question why they are different.

Reply:
done.